internal/app/export/transform.go

```diff
@@ -56,6 +56,35 @@ func TransformScanReport(xml []byte) ([]byte, error) {
 	return out, nil
 }
 
+// TransformProjects adds custom fields and created date to projects
+func TransformProjects(projects []*rest.Project, projectsOData []*rest.ProjectOData) []*rest.Project {
```

Comment:
Please move this into the GetProjects method, which now should return a list of projects that contain the configuration.

The transform functions were created out of the necessity to convert SAST data structures into AST. Using transform functions for building the data returned by api is not in line with that intention.